### DISCUSSION TITLE
Should I install both Local PV hostpath and cStor?

### DISCUSSION THREAD

**Discussion by kamzata** [openebs/openebs#3598] [General] [answered]
I'm just getting my feet wet now with OpenEBS and I'm trying to understand what I should do. I'm planning to use both MongoDB and Mysql databases. [Documentation says](https://openebs.io/docs/#local-volumes) for MongoDB local volumes are preferred because "that are distributed in nature (what does that mean?) and have high availability built into them" while Replicated Volumes are preferred for Mysql.

My questions are:

 - Should I install both Local PV Hostpath and cStor?
 - Is Mayastor ready for production?
 - Why doesn't [Helm Chart OpenOBS page](https://artifacthub.io/packages/helm/openebs/openebs) mention how to install Mayastor?

**Comment by avishnu** [comment, marked as answer]
Sorry for the late responses !!
First of all, great questions.
As the documentation says, CStor and Mayastor are "replicated" storage engines, which store multiple synchronous replicas of the data for redundancy and fail-over.
LocalPV volumes don't support replication.
If the workload or K8s application manages replication at the application level, it is not mandatory to use a replicated storage solution, you could use any of the OpenEBS LocalPV engines. However, It is still desirable to use a replicated solution like Mayastor, since application rebuilds can take up a lot of time.
CStor/Jiva are now EOL and legacy. The latest, stable and most active engine is Mayastor, which is ready for production. You can now install Mayastor from within the OpenEBS helm chart as well.
